core/client/app/routes/setup/one.js

```diff
@@ -41,6 +41,11 @@ export default Ember.Route.extend({
         return DownloadCountPoller.create({url: this.get('ghostPaths.count')});
     },
 
+    setupController: function (controller) {
+        this._super(controller);
+        this.controllerFor('setup').send('toStepOne');
+    },
```

Comment:
`setupController` has two arguments (controller and model) so if you're going to call its super, both arguments really should be supplied (or just use `this._super.apply(this, arguments)`).

I'm not sure why `setupController` is even being used here, though?  It doesn't seem to be the right place to send an action to a completely different controller.